README.md

```diff
@@ -17,11 +17,13 @@ Adaptor consists of two main components:
 
 Both are Java Spring Boot applications, released as separate docker images.
 
-## Developer Information
 
-See [Operating The Patient Switching Adaptor](./OPERATING.md) for Guidance on operating the Patient Switching Adaptor in production.
+## Table of contents
 
-Information for contributors and running the adaptor locally is available in [Developer Information](./developer-information.md).   
+1. [Guidance for operating the adaptor as a New Market Entrant](/OPERATING.md)
```

Comment:
There is 1. against all the points... Shouldn't it be 1, 2, 3... ?

Reply:
Specifying the numbers means that if you want to move around the entries then you have to renumber all the lines. Using 1. means you don't have to.

Reply:
ahh wasn't aware it worked like this.  nice